isis/tests/ClipperNacRollingShutterCameraTests.cpp

```diff
@@ -0,0 +1,55 @@
+#include "ClipperNacRollingShutterCamera.h"
+#include "Fixtures.h"
+#include "TestUtilities.h"
+
+#include <gtest/gtest.h>
+
+using namespace Isis;
+
+TEST_F(ClipperNacCube, ClipperNacRsCameraUnitTest) {
+
+  ClipperNacRollingShutterCamera *cam = (ClipperNacRollingShutterCamera *)testCube->camera();
+
+  EXPECT_EQ(cam->CkFrameId(), -159011);
+  EXPECT_EQ(cam->CkReferenceId(), -159010);
+  EXPECT_EQ(cam->SpkTargetId(), -159);
+  EXPECT_EQ(cam->SpkReferenceId(), 1);
+
+  EXPECT_NEAR(cam->FocalLength(), 150.402, 0.0001);
+
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->spacecraftNameLong(), "Europa Clipper");
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->spacecraftNameShort(), "Clipper");
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentNameLong(), "Europa Imaging System Rolling Shutter Narrow Angle Camera");
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentNameShort(), "EIS-RSNAC");
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentId(), "EIS-NAC-RS");
+
+
+  EXPECT_TRUE(cam->SetImage(145.0, 161.0));
+  EXPECT_NEAR(cam->UniversalLatitude(), 10.248688804675723, 0.0001);
+  EXPECT_NEAR(cam->UniversalLongitude(), 256.15486019464515, 0.0001);
+  EXPECT_TRUE(cam->SetUniversalGround(cam->UniversalLatitude(), cam->UniversalLongitude()));
+  EXPECT_NEAR(cam->Sample(), 290, 0.0001);
+  EXPECT_NEAR(cam->Line(), 322, 0.0001);
```

Comment:
After inputing sample and line to SetImage, the output for sample and line are doubled(for example 145->290). Not sure if this is the expected output after running SetUniversalGround.

Reply:
No it should not be doing that.